pom.xml

```diff
@@ -441,6 +441,7 @@ under the License.
               </filterProperties>
               <environmentVariables>
                 <JENKINS_MAVEN_AGENT_DISABLED>true</JENKINS_MAVEN_AGENT_DISABLED>
+                <_JAVA_OPTIONS>-Dabc=def</_JAVA_OPTIONS>
```

Comment:
Please add a comment behind it like this: `<!-- required for MJAVADOC-538 -->`

Reply:
Added.